data/types/dosingdecision/recommended_basal.go

```diff
@@ -0,0 +1,44 @@
+package dosingdecision
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	MinUnitsPerHour = 0
+	MaxUnitsPerHour = 1000
+	MinDuration     = 0
+	MaxDuration     = 1000
```

Comment:
```suggestion
	MaxDuration     = 604800000
```
Consistent with `BasalScheduled`. Though the units in `BasalScheduled` are milliseconds. Not sure what our units here are...

Reply:
ok